PIL/Image.py

```diff
@@ -570,18 +603,21 @@ def close(self):
 
```

Comment:
In Zulip the approach appears to be to mark the more dynamic cases with simply `# type: ignore`, often with an additional comment afterwards as to the reason; personally I'd suggest doing the same with use of `Any`, so that later searches can find both 'ignore' and 'Any', and find any reasoning in a nearby location.